homework.md

```diff
@@ -50,37 +50,37 @@ A programming language used on the internet
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+It allows you to save multiple files of a project in one place and it allow multiple users to work on the project without affecting the orignal copy without the oners permission.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You saving a copy of the project onto your local drive.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You making a copy of the orignal project.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+It's when you send a copy of your work into git, so that you can commit it to the orignal work so that the owner of the orignal copy can see the changes you made on the project before adding it.
```

Comment:
> owner of the orignal copy can see the changes you made on the project before adding it.
> 
From that line I can clearly tell you have the idea **but** before that line I clearly see a mixing of concepts that have nothing to do with PR (Pull Request)

## Lets break down your answer and let's see where concepts are mixed:

> It's when you send a copy of your work into git
- So the is  one thing that you have to understand that "committing" your file changes will save all your staged changes to your **local** repository **files are not saved into git but your local repo**. Git is a tool you use to save to local and remote repos plus many other task.

> so that you can commit it to the orignal work
- Secondly here you have to understand that you will never commit changes to a repo that you do not have access to but you can only commit to your **forked repo**.
 
### So I would higly suggest you review the following concepts below:
1. PR
1. Commiting
1. Git (What actually it is and does cause I saw a bit of mixing there when you said **"saving changes to git"**)

## If you get lost along the way I am just a Slack message away.